### PR TITLE
EASY-1837: Introducing HSQLDB

### PR DESCRIPTION
**Pull request by rvanheest**
Fixes EASY-1837

#### When applied it will...
* switch from SQLite to HSQLDB for testing purposes

See https://github.com/DANS-KNAW/easy-pid-generator/pull/30 for details

@DANS-KNAW/easy 

@janvanmansum no need to do a release of the parent projects. We'll do that once there are more significant changes.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
 
         <!-- Database connectivity -->
         <postgresql.version>9.4-1205-jdbc42</postgresql.version>
-        <sqlite.version>3.21.0</sqlite.version>
```
Comment (janvanmansum) [inline pom.xml]: We should keep the SQLite config for a while (next to the hsqldb config), just in case some project is still using it.

**Review by janvanmansum — CHANGES_REQUESTED** [review]
See comment.

File: pom.xml (hunk 1 above)
Comment (rvanheest) [reply pom.xml]: @janvanmansum do you only want to keep the `<sqlite.version>3.21.0</sqlite.version>` or also the dependency that was replaced below?

Comment (rvanheest) [reply pom.xml]: see 87cb2c3

**Pull request merged**